opal/mca/btl/openib/btl_openib_component.c

```diff
@@ -2635,23 +2635,10 @@ btl_openib_component_init(int *num_btl_modules,
         goto no_btls;
     }
 
-    /* If we want fork support, try to enable it */
-#ifdef HAVE_IBV_FORK_INIT
-    if (0 != mca_btl_openib_component.want_fork_support) {
-        if (0 != ibv_fork_init()) {
-            /* If the want_fork_support MCA parameter is >0, then the
-               user was specifically asking for fork support and we
-               couldn't provide it.  So print an error and deactivate
-               this BTL. */
-            if (mca_btl_openib_component.want_fork_support > 0) {
-                opal_show_help("help-mpi-btl-openib.txt",
-                               "ibv_fork_init fail", true,
-                               opal_process_info.nodename);
-                goto no_btls;
-            }
-        }
+    /* If fork support is requested, try to enable it */
```

Comment:
Creating a synonym here meant to have a cross-components synonym. (openib btl and opal). I think that setting btl_openib_want_fork_support shouldn't affect all other components which would happen if there were synonyms.
The opal mca parameter controls all the components as implied by its name.
Also, I'm not sure that this openib btl was widely used since setting in to 1 (before this commit) and asking specifically for btl openib, failed. (because of the late call to ibv_fork_init)